scapy/arch/bpf/supersocket.py

```diff
@@ -208,11 +207,15 @@ def close(self):
 
     def send(self, x):
         """Dummy send method"""
-        raise Exception("Can't send anything with %s" % self.__name__)
+        raise Exception(
+            "Can't send anything with %s" % self.__class__.__name__
+        )
 
-    def recv(self, x=BPF_BUFFER_LENGTH):
+    def recv_raw(self, x=BPF_BUFFER_LENGTH):
         """Dummy recv method"""
-        raise Exception("Can't recv anything with %s" % self.__name__)
+        raise Exception(
```

Comment:
unnecessary change.

Reply:
It's not.
`.__name__` doesn't exist on an instance.
However, `.__class__.__name__` does